.github/workflows/CI.yml

```diff
@@ -0,0 +1,25 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.15
```

Comment:
Is 1.16 not available yet?

Reply:
How about this? 

```
jobs:
  test-build:
    name: Test & Build
    strategy:
      matrix:
        go-version: [1.15.x, 1.16.x]
        platform: [ubuntu-latest, macos-latest, windows-latest]
    runs-on: ${{ matrix.platform }}
    steps:
    - name: setup go
      uses: actions/setup-go@v2
      with:
        go-version: ${{ matrix.go-version }}
      id: go

```

Reply:
Added 1.16.
Thanks for letting me know about matrices in github actions, I have added them.
Do we need the `id: go` part?

Reply:
not sure, I copied it from somewhere else :)